### PR TITLE
pipelines: explorer

### PR DESCRIPTION
**Pull request by eedygreen**
## Description
Mainnet Pipelines for Explorer Indexer and API

### CODE REVIEW

File: .github/workflows/deploy_mainnet_api.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+       uses: cuchi/jinja2-action@v1.2.0
+       with:
+         template: 'sygma-explorer/ecs/task_definition-explorer-api-${{ env.ENVIRONMENT }}.j2'
+         output_file: 'sygma-explorer/ecs/task_definition-explorer-api-${{ env.ENVIRONMENT }}.json'
+         data_format: json
+         variables: |
+           awsAccountId=${{ env.AWS_MAINNET }}
+           awsRegion=${{ env.AWS_REGION }}
+           DB_USERNAME=${{ secrets.MAIN_USERNAME }}
+           DB_PASSWROD=${{ secrets.MAIN_PASSWORD }}
```
Comment (mpetrun5) [inline .github/workflows/deploy_mainnet_api.yml:49]:
```suggestion
           DB_PASSWORD=${{ secrets.MAIN_PASSWORD }}
```

File: .github/workflows/deploy_mainnet.yml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+       uses: cuchi/jinja2-action@v1.2.0
+       with:
+         template: 'sygma-explorer/ecs/task_definition-explorer-indexer-${{ env.ENVIRONMENT }}.j2'
+         output_file: 'sygma-explorer/ecs/task_definition-explorer-indexer-${{ env.ENVIRONMENT }}.json'
+         data_format: json
+         variables: |
+           awsAccountId=${{ env.AWS_MAINNET }}
+           awsRegion=${{ env.AWS_REGION }}
+           DB_USERNAME=${{ secrets.MAIN_USERNAME }}
+           DB_PASSWROD=${{ secrets.MAIN_PASSWORD }}
```
Comment (mpetrun5) [inline .github/workflows/deploy_mainnet.yml:49]:
```suggestion
           DB_PASSWORD=${{ secrets.MAIN_PASSWORD }}
```

File: .github/workflows/deploy_mainnet.yml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+     - name: Configure AWS Credentials
+       uses: aws-actions/configure-aws-credentials@v1
+       with:
+         role-to-assume: arn:aws:iam::${{ env.AWS_MAINNET }}:role/github-actions-${{ env.ENVIRONMENT }}-chainbridge
```
Comment (mpetrun5) [inline .github/workflows/deploy_mainnet.yml:54]:
```suggestion
         role-to-assume: arn:aws:iam::${{ env.AWS_MAINNET }}:role/github-actions-${{ env.ENVIRONMENT }}-chainbridge
```

Shouldn't this be sygma?

Comment (mpetrun5) [reply .github/workflows/deploy_mainnet.yml:54]: Same for api.

**Review by mpetrun5 — CHANGES_REQUESTED** [review]

**Review by MakMuftic — APPROVED** [review]

**Review by mpetrun5 — APPROVED** [review]

**Pull request merged**